orca/orca-web/src/main/groovy/com/netflix/spinnaker/orca/web/config/WebConfiguration.groovy

```diff
@@ -67,6 +69,18 @@ class WebConfiguration {
     OrcaObjectMapper.getInstance()
   }
 
+  @Bean(name = "objectMapper") ObjectMapper orcaObjectMapper(Optional<SerializerHookRegistry> serializerModifier) {
```

Comment:
Isn't this now a duplicate bean from the core and not needed?  OR am I missing something?

Reply:
Yep! I realized that we did not have the `@Bean{ mapper, objectMapper}` it was just `mapper`. So yea, this can go away!